lib/mayaUsd/python/wrapUtil.cpp

```diff
@@ -56,14 +56,16 @@ std::string ensureUSDFileExtension(const std::string& fileToCheck)
 
 void wrapUtil()
 {
-    scope s = class_<UsdMayaUtilScope>("Util", no_init)
-                  .def("IsAuthored", UsdMayaUtil::IsAuthored)
-                  .def("prettifyName", &UsdMayaUtil::prettifyName)
-                  .staticmethod("prettifyName")
-                  .def("getDictionaryFromEncodedOptions", getDictionaryFromEncodedOptions)
-                  .def(
-                      "getPathRelativeToMayaSceneFile",
-                      UsdMayaUtilFileSystem::getPathRelativeToMayaSceneFile)
-                  .def("ensureUSDFileExtension", ensureUSDFileExtension)
-                  .staticmethod("getPathRelativeToMayaSceneFile");
+    scope s
+        = class_<UsdMayaUtilScope>("Util", no_init)
+              .def("IsAuthored", UsdMayaUtil::IsAuthored)
```

Comment:
For the life of me, I cannot figure out why clang-format decided all this needed to be reformatted. The rules are so obscure!